Library/Formula/libsearpc.rb

```diff
@@ -0,0 +1,40 @@
+class Libsearpc < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/libsearpc/archive/v3.0-latest.tar.gz"
+  sha1 "cdff106886441205f46a592100ae324314127107"
+  revision 1
+
+  head "https://github.com/haiwen/libsearpc.git"
+
+  #FIX for homebrew autotools path
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!